src/main/java/com/xiaomi/infra/pegasus/client/PegasusClientInterface.java

```diff
@@ -58,11 +58,12 @@
    *     administrator
    * @param backupRequestDelayMs the delay time to send backup request. If backupRequestDelayMs <=
    *     0, The backup request is disabled.
+   * @param enableCompression whether to enable the table data open auto-compress
    * @return the table handler
    * @throws PException throws exception if any error occurs.
    */
-  public PegasusTableInterface openTable(String tableName, int backupRequestDelayMs)
-      throws PException;
+  public PegasusTableInterface openTable(
+      String tableName, int backupRequestDelayMs, boolean enableCompression) throws PException;
```

Comment:
```suggestion
  public PegasusTableInterface openTable(
      String tableName, int backupRequestDelayMs, boolean enableCompression) throws PException;
```

I think TableOptions should be exposed to the users. Your change on this API gonna break compatibility with the previous version.

Reply:
I add `openTable(String tableName, TableOptions tableOptions)` and retain origin Interface but mark `Deprecated`